manifest.json

```
@@ -2,15 +2,15 @@
   "name": "Counter",
   "author": "Placeholder-author",
   "description": "An application for Aragon",
-  "details_url": "/dist/meta/details.md",
+  "details_url": "/meta/details.md",
   "source_url": "https://<placeholder-repository-url>",
-  "icons": [{
-    "src": "/dist/meta/icon.svg",
-    "sizes": "56x56"
-  }],
-  "screenshots": [
-    { "src": "/dist/meta/screenshot-1.png" }
+  "icons": [
+    {
+      "src": "/meta/icon.svg",
+      "sizes": "56x56"
+    }
   ],
-  "start_url": "/dist/index.html",
-  "script": "/dist/script.js"
+  "screenshots": [{ "src": "/meta/screenshot-1.png" }],
+  "start_url": "index.html",
+  "script": "script.js"
```

```suggestion
  "script": "/script.js"
```

I don't think it matters (I think the client does this automatically), but might as well stay consistent with the other asset files.